src/payment/data/sagas.js

```diff
@@ -165,6 +175,26 @@ export function* handleFetchCaptureKey() {
   }
 }
 
+/**
+ * Redux saga for getting hte client secret key for a Stripe payment
+ */
+export function* handleFetchClientSecret() {
+  if (yield isClientSecretProcessing()) {
+    return;
+  }
+
+  try {
+    yield put(clientSecretProcessing(true));
+    // TODO: possibly add status for stripe elements loading?
+    // const result = yield call(PaymentApiService.getClientSecret);
```

Comment:
Is this comment intentional?
I believe you're missing updating the store with the client secret data